### PR TITLE
ts tests

### PR DESCRIPTION
**Pull request by zjb0807**

### CODE REVIEW

File: runtime/karura/src/lib.rs [hunk 1]
```diff
@@ -1299,6 +1303,20 @@ pub type ScheduleCallPrecompile = runtime_common::ScheduleCallPrecompile<
...
 
+pub struct FindAuthorTruncated<F>(PhantomData<F>);
+impl<F: FindAuthor<u32>> FindAuthor<H160> for FindAuthorTruncated<F> {
+	fn find_author<'a, I>(digests: I) -> Option<H160>
+	where
+		I: 'a + IntoIterator<Item = (ConsensusEngineId, &'a [u8])>,
+	{
+		if let Some(author_index) = F::find_author(digests) {
+			let authority_id = Aura::authorities()[author_index as usize].clone();
+			return Some(H160::from_slice(&authority_id.to_raw_vec()[4..24]));
```
Comment (xlc) [inline runtime/karura/src/lib.rs:1314]: should just use our EVM account mapping to convert Substrate address to EVM address

Comment (xlc) [reply runtime/karura/src/lib.rs:1314]: we can use the standard `FindAuthor<AccoutnId>` and do the conversion in module_evm

File: ts-tests/build/IERC20Metadata.json [hunk 2]
```diff
@@ -0,0 +1,244 @@
+{
```
Comment (xlc) [inline ts-tests/build/IERC20Metadata.json:1]: do we want to include the build folder?

Comment (zjb0807) [reply ts-tests/build/IERC20Metadata.json:1]: Prepare to delete the `modules/evm-bridge/src/erc20_demo_contract` file and reference it from the build directory, but there is a problem

**Review by xlc — APPROVED** [review]

**Pull request merged**